Tasks/AzureVmssDeployment/task.json

```diff
@@ -33,6 +29,17 @@
             "displayName": "Image Details",
             "isExpanded": true,
             "visibleRule": "action = Update image || action = UpdateImage"
+        },
+        {
+            "name": "StartupConfiguration",
+            "displayName": "Configure start-up",
+            "isExpanded": true,
+            "visibleRule": "action = Configure application startup || action = Update image || action = UpdateImage"
```

Comment:
why was the change  made. Since it was an internal value - we should avoid that. Can you look at the usage and if there is not much - consider removing the older one

Reply:
this is not exactly an internal value. this is used to generate task instance name. and that's why it was changed to make it more friendly.